internal/resources/webform/webform.js

```diff
@@ -1923,7 +1923,7 @@ window.initGRPCForm = function(services, invokeURI, metadataURI, debug, headers)
             case "google.protobuf.BoolValue":
                 return false;
             case "google.protobuf.Value":
-                return null;
+                return "";
```

Comment:
I think `null` is a better default than empty string (as would be `{}`). Also, `null` is an acceptable value for a `google.protobuf.Value`. So perhaps the real issue is somewhere else, not properly handling the `null` value?

Reply:
yes, i will debug to see where it's not handling the null value

Reply:
we can made an if inside `addOptionalMessage` to check if `fld` is `google.protobuf.Value` to then return `addElementToForm(schema, container, parent, pathLen, value, allowMissing, fld);`

Reply:
if you agree with this solution or the other one, please tell me, than i will fix

Reply:
i already test it with the if and it works